src/components/checkout/Checkout.jsx

```
@@ -209,7 +215,15 @@ const Checkout = () => {
     }
   };
 
-  const { register, handleSubmit, watch, getValues, setValue, setError, formState: { errors } } = useForm({
+  const {
+    register,
+    handleSubmit,
+    watch,
+    getValues,
+    setValue,
+    setError,
+    formState: { errors },
```

ESLint

[error] 'errors' is assigned a value but never used.

(no-unused-vars)